hq/app/logic/IamRemediation.scala

```diff
@@ -40,9 +41,24 @@ object IamRemediation extends Logging {
     * Looks through the credentials report to identify users with Access Keys that are older than we allow.
     */
   def identifyUsersWithOutdatedCredentials(awsAccount: AwsAccount, credentialReportDisplay: CredentialReportDisplay, now: DateTime): List[IAMUser] = {
-    ???
+    credentialReportDisplay.machineUsers.filter(user => hasOutdatedMachineKey(List(user.key1, user.key2), now)).toList ++
+      credentialReportDisplay.humanUsers.filter(user => hasOutdatedHumanKey(List(user.key1, user.key2), now))
   }
 
+  private def hasOutdatedHumanKey(keys: List[AccessKey], now: DateTime): Boolean = keys.exists { key =>
```

Comment:
In the future, we might want to think about unifying these implementations with the ones that already exist for the UI:
https://github.com/guardian/security-hq/blob/main/hq/app/logic/VulnerableAccessKeys.scala#L8

No worries for now though!